Misc/NEWS.d/next/Core and Builtins/2023-03-01-01-36-39.gh-issue-102344.Dgfux4.rst

```diff
@@ -0,0 +1,2 @@
+Implement ``winreg.QueryValue`` using ``QueryValueEx`` and
+``winreg.SetValue`` using ``SetValueEx``. Patch by Max Bachmann.
```

Comment:
Let's move this NEWS entry into the "Windows" section (instead of "Core and Builtins"). Just a file move is fine, the name can stay the same.

Reply:
done